File: scripts/simple_fix.py

import os

def main():
    core_path = os.path.join(os.path.dirname(__file__), '..', 'js', 'core.js')
    
    # Read as binary first
    with open(core_path, 'rb') as f:
        raw = f.read()
    
    # Decode as utf-8
    content = raw.decode('utf-8')
    
    # Make a copy
    new_content = content
    
    # Replace only the specific patterns we see in the user's screenshot
    # Based on the grep results from earlier
    replacements = [
        ('娴忚?', '浏览'),
        ('鐐硅禐', '点赞'),
        ('璇勮?', '评论'),
        ('鍒犻櫎', '删除'),
        ('鏌ョ湅璧勬枡', '查看资料'),
        ('鏈夊鐧诲綍', '未登录'),
        ('鐐瑰嚮鐧诲綍', '点击登录'),
        ('涓炬姤', '举报'),
    ]
    
    for old, new in replacements:
        count = new_content.count(old)
        if count > 0:
            new_content = new_content.replace(old, new)
            print(f"Replaced {count} instances of {repr(old)} with {repr(new)}")
    
    # Write back carefully
    with open(core_path, 'w', encoding='utf-8') as f:
        f.write(new_content)
    
    print('File written successfully!')

if __name__ == '__main__':
    main()

File: scripts/simple_fix.js
const fs = require('fs');
const path = require('path');

console.log('开始修复 core.js！');

const corePath = path.join(__dirname, '..', 'js', 'core.js');
let content = fs.readFileSync(corePath, 'utf8');

// 简单的修复，处理最关键的几个乱码:
const fixes = [
    ['鐐硅禐', '点赞'],
    ['璇勮?', '评论'],
    ['鎶?鍙?', '举报'],
    ['娴忚?', '浏览'],
    ['鏌ョ湅璧勬枡', '查看资料'],
    ['鏈夊鐧诲綍', '未登录'],
    ['鐐瑰嚮鐧诲綍', '点击登录'],
    ['涓炬姤', '举报'],
    ['鏈嶅姟鍔犺浇澶辫触锛岃鍒锋柊椤甸潰閲嶈瘯', '服务加载失败，请刷新页面重试'],
];

let total = 0;
for (const [from, to] of fixes) {
    const regex = new RegExp(from.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'g');
    const matches = content.match(regex);
    if (matches) {
        total += matches.length;
        content = content.replace(regex, to);
        console.log(`修复了 ${from} → ${to} (${matches.length}处)`);
    }
}

if (total > 0) {
    fs.writeFileSync(corePath, content, 'utf8');
    console.log(`\n✅ 成功！共修复了 ${total} 处乱码！`);
} else {
    console.log('未找到需要修复的内容...');
}
